purposes-archiver/src/main/resources/application.conf

```diff
@@ -0,0 +1,19 @@
+include "akka"
+
+services {
+    agreement-process               = ${AGREEMENT_PROCESS_URL}
+    purpose-process                 = ${PURPOSE_PROCESS_URL}
+    purposes-archiver-queue-name    = ${PURPOSES-ARCHIVER_QUEUE_NAME}
+    visibility-timeout-in-seconds   = 60
+    visibility-timeout-in-seconds   = ${?QUEUE_VISIBILITY_TIMEOUT_IN_SECONDS}
```

Comment:
just to keep things in order, under `services` we should put only configurations of other services (urls, keys, etc)